lib/node_modules/@stdlib/stats/base/dnanmeanors/README.md

```diff
@@ -181,6 +176,107 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dnanmeanors.h"
+```
+
+#### stdlib_strided_dnanmeanors( N, \*X, strideX )
+
+Computes the arithmetic mean of a double-precision floating-point strided array, ignoring `NaN` values and using ordinary recursive summation.
+
+```c
+const double x[] = { 1.0, 2.0, 0.0/0.0, 3.0, 0.0/0.0, 4.0, 5.0, 6.0, 0.0/0.0, 7.0, 8.0, 0.0/0.0 };
+
+double v = stdlib_strided_dnanmeanors( 6, x, 2 );
+// returns 4.6667
+```
+
+The function accepts the following arguments:
+
+-   **N**:       `[in] CBLAS_INT` number of indexed elements.
+-   **X**:       `[in] double*` input array.
+-   **strideX**: `[in] CBLAS_INT` stride length for `X`.
+
+```c
+double stdlib_strided_dnanmeanors( const CBLAS_INT N, const double *X, const CBLAS_INT strideX );
+```
+
+#### stdlib_strided_dnanmeanors_ndarray( N, \*X, strideX, offsetX )
+
+Computes the arithmetic mean of a double-precision floating-point strided array, ignoring `NaN` values and using ordinary recursive summation and alternative indexing semantics.
+
+```c
+const double x[] = { 1.0, 2.0, 0.0/0.0, 3.0, 0.0/0.0, 4.0, 5.0, 6.0, 0.0/0.0, 7.0, 8.0, 0.0/0.0 };
+
+double v = stdlib_strided_dnanmeanors( 6, x, 2, 0 );
+// returns 4.6667
+```
+
+The function accepts the following arguments:
+
+-   **N**:       `[in] CBLAS_INT` number of indexed elements.
```

Comment:
```suggestion
-   **N**: `[in] CBLAS_INT` number of indexed elements.
```

Don't align this in READMEs. This needs to be fixed here and elsewhere in this file.